add_new_dc_test.py

```diff
@@ -69,6 +73,7 @@ def reconfigure_keyspaces_to_use_network_topology_strategy(self, keyspaces: List
             node.run_cqlsh(cql)
         self.log.info("Replication Strategies for {} reconfigured".format(keyspaces))
 
+    @optional_stage('prepare_write')
```

Comment:
`@optional_stage` -> `test_stage`

Reply:
I was having doubts if to change this one. `optional_stage` name is assuming some alternative - i.e. just from decorator name it can be read that the decorated function can be skipped (as it is optional).
`test_stage` is not clear from this perspective. Maybe something as `skip_if_test_stage_disabled` would be approrpiate?

Reply:
ok